odl/operator/oputils.py

```diff
@@ -178,6 +180,69 @@ def power_method_opnorm(op, niter, xstart=None):
         op(x, out=tmp)
         return tmp.norm()
 
+
+def as_scipy_operator(op):
+    """ Wrap ``op`` as a ``scipy.sparse.linalg.LinearOperator``
+
+    This is intended to be used with the scipy sparse linear solvers.
+
+    Parameters
+    ----------
+    op : `Operator`
+        A linear operator that should be wrapped
+
+    Returns
+    -------
+    `scipy.sparse.linalg.LinearOperator` : linear_op
```

Comment:
If this link works, you could use single backticks in the one-line summary, too.